etcdmgmt/etcd-mgmt.go

```diff
@@ -33,6 +33,9 @@ var (
 	ExecName = "etcd"
 	// Configuration directory for storing etcd configuration
 	ETCDConfDir = "/var/lib/glusterd"
+	// Etcd -data-dir location. This contains WAL files and log snapshots
+	// https://coreos.com/etcd/docs/latest/admin_guide.html#data-directory
+	ETCDDataDir = "/var/lib/etcd"
```

Comment:
We don't yet support an externally managed etcd yet, but it is on the TODO list.
The externally managed etcd, if it has a server running on a GD2 host, will have it's conf/store as per the conventions in the etcd package.

Currently we only support etcd managed by ourselves. For this sort of an etcd, it's my opinion that, its conf and data should be under the GD2 paths. This managed etcd should ideally not conflict with any other etcd daemons running on the system. We would be clashing ports with other etcds, and that is something that we should change.